### PR TITLE
Feature/ecs service

### PR DESCRIPTION
**Pull request by tobafolaranmi-ch**
## Description

Please explain briefly the changes you made here.

## Jira Ticket

Please add link to Jira ticket

## Coding Standards

Code Style Guide: https://github.com/companieshouse/dissolution-web/wiki/Code-Style-Guide

## Checklist

- [ ] 3 Amigos
- [ ] Acceptance Criteria met
- [ ] Branch named {feature|hotfix|task}/{S4-*}
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] All unit tests passing
- [ ] WAVE accessibility testing tool ran on new or updated pages
- [ ] All UI Tests Passing
- [ ] Pulled latest master into feature branch
- [ ] Sonar Analysis
- [ ] Master branch on Rebel1 and CIDev CI/CD pipeline is green
- [ ] Any Docker environment variables added are consistent with those on our upstream environments (Rebel1, CIDev etc)
- [ ] If your pull request depends on any other, please link them in the description

### CODE REVIEW

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# cache_server 
+# oauth2_redirect_uri 
+# oauth2_auth_uri 
+# account_url 
+log_level = "debug"
+chs_url = "https://cidev.aws.chdev.org"
+piwik_url = "https://matomo.platform.aws.chdev.org/"
+piwik_site_id = "7"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+chips_presenter_auth_url = "http://enablement-presenter-api:8080/presenterauth"
```
Comment (bcullerton) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:20]: This looks like what could be a live value to be, does it need to make reference to cidev anywhere?

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+account_local_url = "https://account.cidev.aws.chdev.org"
+pay_by_account_feature_enabled = "1"
+piwik_confirmation_page_pdf_goal_id = "998"
+piwik_landing_page_start_goal_id = "999"
+piwik_limited_company_goal_id = "999"
+piwik_partnership_goal_id = "999"
+piwik_single_director_confirmation_goal_id = "1000"
+piwik_multi_director_confirmation_goal_id = "1001"
+piwik_limited_company_confirmation_goal_id = "1002"
+piwik_partnership_confirmation_goal_id = "1003"
```
Comment (bcullerton) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:35]: Some of these values are also different to cidev based on the env configs here - https://github.com/companieshouse/chs-configs/blob/develop/cidev/dissolution-web/env

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# account_url 
+log_level = "debug"
+chs_url = "https://cidev.aws.chdev.org"
+piwik_url = "https://matomo.platform.aws.chdev.org/"
+piwik_site_id = "7"
+cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
+chips_presenter_auth_url = "http://enablement-presenter-api:8080/presenterauth"
+chs_company_profile_api_local_url = "https://api.cidev.aws.chdev.org"
+cache_db = "0"
+dissolutions_api_url = "https://api.cidev.aws.chdev.org"
```
Comment (bcullerton) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:23]: Aware we've discussed this already but this is a global var in the mesos setup, worth replicating?

**Review by bcullerton — APPROVED** [review]

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 3 above)
Comment (bcullerton) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:23]: Covered already in conversation on node-18-upgrade

**Review by bcullerton — CHANGES_REQUESTED** [review]
Just some corrections to the terraform cidev vars required

**Review by bcullerton — APPROVED** [review]

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 1 above)
Comment (tobafolaranmi-ch) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars]: (https://github.com/companieshouse/chs-configs/blob/develop/cidev/dissolution-web/env)
Value here is 
export CHIPS_PRESENTER_AUTH_URL=http://localhost:18211/presenterauth
I'll make that change

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars (hunk 3 above)
Comment (tobafolaranmi-ch) [reply terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:26]: Yup, we're discussing setting up global vars lookup

**Pull request merged**